rancher/distros-test-framework
# K3s split role

#### Proposed Changes ####



Adding support to be able to deploy k3s with split role. 
Changes to terraform files.
Added k3s_node_role.sh script. 
minor update to k3s_master.sh for the etcd only node as master node scenario. 
updates to files in modules/k3s:
instances_server.tf file
main.tf
variables.tf



#### Types of Changes ####



#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

2. Verify code lint; we should not have errors.
done

3. Update the documentation if needed.
docs examples already have k3s examples. nothing additional needed

4. Update makefile and docker run if adding new tests.
n/a

5. Run your tests at least 4 times with all configurations needed and possible.
done

6. If needed test with different os types.
n/a

#### Linked Issues ####



#### Further Comments ####
